libfwupd/fwupd-context-test.c

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (C) 2020 Philip Withnall <philip@tecnocode.co.uk>
```

Comment:
```suggestion
 * Copyright (C) 2020 Philip Withnall <pwithnall@endlessos.org>
```

(I did it on work time)